service/src/main/kotlin/fi/espoo/evaka/application/ApplicationQueries.kt

```diff
@@ -857,7 +857,10 @@ fun removeOldDrafts(db: Database.Transaction, deleteAttachment: (db: Database.Tr
 fun Database.Transaction.cancelOutdatedTransferApplications(): List<UUID> = createUpdate(
     """
 UPDATE application SET status = :cancelled
-WHERE transferapplication AND NOT EXISTS (
+WHERE transferapplication
+-- only include applications that don't have decisions
```

Comment:
Comments in SQL are problematic with the current JDBI version because its parser does not understand them...it might work or might not work, but I recommend not having any comments. I don't remember the exact details but we had one at some point a query that did a wrong thing because of some SQL comments that confused the parser.

JDBI 3.18.1 is supposed to fix this...